import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/LocalBackupUploadService.java

```diff
@@ -30,7 +31,10 @@ public void doWriteBackup(byte[] backupData, String uploadFilePath) throws Backu
     }
 
     @Override
-    public void removeBackup(String uploadFilePath) {
-        // todo: implement_me
+    public void removeBackup(String uploadFilePath) throws BackupExeption {
+        File file = new File(uploadFilePath);
+        if (!file.delete()){
+            throw new BackupExeption("Error with removing temporary file", new Exception());
```

Comment:
BackupExeption it`s own Service ex. I y need BackupExeption(msg) constructor - then create it.